anchor/network/src/discovery.rs

```diff
@@ -304,14 +364,42 @@ impl Discovery {
                                 (enr, None)
                             })
                             .collect();
+                        debug!(peers = ?results, "Discovery query completed");
                         return Some(results);
                     }
                     Err(e) => {
                         warn!(error = %e, "Discovery query failed");
                     }
                 }
             }
-            _ => {
+            QueryType::Subnet(queries) => {
+                let subnets_searched_for: Vec<SSVSubnet> =
+                    queries.iter().map(|query| query.subnet).collect();
+
+                match query.result {
+                    Ok(r) if r.is_empty() => {
+                        debug!(
+                            subnets_searched_for = ?subnets_searched_for,
+                            "Grouped subnet discovery query yielded no results.",
+                        );
+                        // TODO queries.iter().for_each(|query| {
```

Comment:
I was referring to these todos

Reply:
It's because it's based on LH code and there're more things that I don't understand yet and I'm not sure it's necessary now. What do you think?